README.md

```diff
@@ -82,13 +82,34 @@ The application is deployed to the following URLs:
 
 | Environment | URL |
 |:------------|:----|
-| dev     | https://tta-smarthub.app.cloud.gov/ |
-| staging | |
+| sandbox | https://tta-smarthub-sandbox.app.cloud.gov/ |
+| dev     | https://tta-smarthub-dev.app.cloud.gov/ |
+| staging | https://tta-smarthub-staging.app.cloud.gov/ |
 | prod    | |
 
+**Sandbox Environment**
+
+An additional deployment environment called "sandbox" is available to developers for testing and
+development on feature branches prior to making a commit to Ad Hoc's main branch and deploying to "dev".
+The current [.circleci/config.yml](.circleci/config.yml) assumes deployments to this space are being made from
+the Ad Hoc repository. Deployments are pushed to the ohstta-sandbox cloud.gov space. To conserve
+resources, running application instances pushed to this space should be deleted as soon as they are no longer needed.
+Running application instances can be deleted by logging into [cloud.gov][cloudgov], and then selecting and
+deleting the application.
+
+**Secret Management**
+
+CircleCI's [project-based environment variables][env-vars] are used for secret management. These secrets include:
+- Cloud.gov deployer account username and password. These keys are specific to each cloud.gov
+organization/space, i.e. each deployment environment, and can be [regenerated by developers][cloudgov-deployer]
+who have proper cloud.gov permissions at any time.
+- HSES authentication middleware secrets passed to the application as `AUTH_CLIENT_ID` and `AUTH_CLIENT_SECRET`.
+- The application `SESSION_SECRET`.
 
 <!-- Links -->
 
 [adhoc-main]: https://github.com/adhocteam/Head-Start-TTADP/tree/main
+[cloudgov]: https://dashboard.fr.cloud.gov/home
+[cloudgov-deployer]: https://cloud.gov/docs/services/cloud-gov-service-account/
```

Comment:
Love that this got documented in the README. I end up googling for this page every time I set up a new space.